products/looker.md

```diff
@@ -3,15 +3,15 @@ title: Looker
 permalink: /looker
 category: server-app
 iconSlug: looker
-releasePolicyLink: https://docs.looker.com/relnotes/release-overview
+releasePolicyLink: https://cloud.google.com/looker/docs/release-overview
 changelogTemplate: |
-  https://docs.looker.com/relnotes/v{{"__RELEASE_CYCLE__" | split:'.' | first}}-changelog#{{"__RELEASE_CYCLE__"}}
+  https://cloud.google.com/looker/docs/looker-{{"__RELEASE_CYCLE__" | split:'.' | first}}-changelog#{{"__RELEASE_CYCLE__" | replace:'.',''}}
 LTSLabel: "<abbr title='Extended Support Release'>ESR</abbr>"
 eolColumn: Support Status
 activeSupportColumn: false
 releaseDateColumn: true
 releaseColumn: false
```

Comment:
because of this one is false , links doesnt even show , so this change is abit trivial if you wont change this to true

Reply:
You are right, I thought the column did not show because there was no supported version (so I opened #1836).

But even if this link is not used, does it means it does not have to be updated ?

Reply:
no i mean just make it true so we can see links :D

Reply:
Let me try if it wont work i will redo it

Reply:
yep you are right lets we fill it with dummy values ( exactly same with its releasecycle )

Reply:
yep this fixes but to be honest i forget to check if any of them non eol release
so i will remove both 2 commits ( dont worry we squash first )
then i will merge

Reply:
@marcwrobel is this fixed that ? or needs another pr for it ?

Reply:
Another PR will be needed : last version on https://endoflife.date/looker is 22.8, but https://cloud.google.com/looker/docs/officially-supported-releases mention a bunch of new versions (up to 22.20).